meteor/server/api/ingest/commit.ts

```diff
@@ -413,7 +486,6 @@ function canRemoveSegment(
 		(nextPartInstance?.segmentId === segmentId && isTooCloseToAutonext(currentPartInstance, false))
 	) {
 		// Don't allow removing an active rundown
-		logger.warn(`Not allowing removal of current playing segment "${segmentId}", making segment unsynced instead`)
```

Comment:
Why is this being removed?

Reply:
It has been moved to here, as `canRemoveSegment` is used in multiple places where that message isn't accurate https://github.com/nrkno/tv-automation-server-core/blob/4fa9cc1cebc8fe896f3b8dbcfbc27414060e779b/meteor/server/api/ingest/commit.ts#L222
e.g. it is used here https://github.com/nrkno/tv-automation-server-core/blob/4fa9cc1cebc8fe896f3b8dbcfbc27414060e779b/meteor/server/api/ingest/commit.ts#L270